doc/whats_new/v0.22.rst

```diff
@@ -421,6 +421,11 @@ Changelog
 - |Efficiency| :meth:`impute.MissingIndicator.fit_transform` avoid repeated
   computation of the masked matrix. :pr:`14356` by :user:`Harsh Soni <harsh020>`.
 
+- |Fix| Fixed a bug in :class:`impute.IterativeImputer` where features where
+  imputed in the reverse desired order with ``imputation_order`` either
+  ``"ascending"`` or ``"descending"``. :pr:`15393` by :user:
+  `Venkatachalam N <venkyyuvy>`.
```

Comment:
This ` needs to be right after the preceding :.

Reply:
@adrinjalali do you want to be credited here?

Reply:
> @adrinjalali do you want to be credited here?

No I didn't find or fix it anyway ;)

Reply:
You are too humble man :)

Reply:
:)

This needs to be

```
  ``"ascending"`` or ``"descending"``. :pr:`15393` by
  :user:`Venkatachalam N <venkyyuvy>`.
```

Reply:
Sure, updated now. Thanks for the corrections.